src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -569,10 +570,12 @@ def _uploadJobDependencies(self, arcJobID, executableFile, inputs):
             self.log.verbose("Input correctly uploaded", fileToSubmit)
         return S_OK()
 
-    def submitJob(self, executableFile, proxy, numberOfJobs=1, inputs=None, outputs=None):
+    def submitJob(self, executableFile, proxy, numberOfJobs=1, inputs=None, outputs=None, diracXSecrets=[]):
```

Comment:
I wonder if it would make sense to generalize that to `env`, being a `list[dict[str, str]]`, so that one can provide additional environment variables to the submission scripts.

```suggestion
    def submitJob(self, executableFile, proxy, numberOfJobs=1, inputs=None, outputs=None, env=[]):
```

For instance, you could have:

```python
env=[{"DIRACX_SECRET": "1234", "SOMETHING_ELSE": "hello"}, {"DIRACX_SECRET": "4567"}]
```

In this way we don't add too many pilot-specific elements in the CEs (even though they already contain many WMS-specificities)